ember-app/app/controllers/issue.js

```diff
@@ -1,7 +1,15 @@
 import Ember from 'ember';
+import IssueEvent from "app/mixins/events/issue";
+import Messaging from "app/mixins/messaging";
 
-var IssueController = Ember.Controller.extend({
+var IssueController = Ember.Controller.extend(
+  IssueEvent, Messaging, {
   needs: ["application"],
+
+  //Subscribes to Messages in the Route after the model is set
+  subscribeDisabled: true,
```

Comment:
When the subscriber is parsing the events it looks up the identifier binding, which is based on the issue controller model. Because of the way we are instantiating this particular controller it needs to wait until the model is present before the subscriptions are set.